detect_secrets/plugins/common/initialize.py

```diff
@@ -137,7 +137,7 @@ def _remove_key(d, key):
         try:
             plugins_dict[plugin_name][param_name] = param_value
         except KeyError:
-            log.warning(
+            log.debug(
```

Comment:
I changed this to debug (will only display with `--verbose` option). If it's logged as a warning I always see this when running a scan even though the `KeywordDetector` plugin is not in the baseline file for this repo:
```
--keyword-exclude specified, but KeywordDetector not configured! Ignoring...
```

I tried updating the baseline file so it would include the keyword detector, but this broke the build because the `pre-commit` hook step within the build auto-updates the baseline file 🤷‍♀️.